…AdministrativeInformation (#699)
admin-shell-io/aas-specs: documentation/IDTA-01001/modules/ROOT/pages/spec-metamodel/common.adoc

Data specifications are defined in IDTA-01003 sub-series.

If an AAS contains two different Submodels with the same semanticId 
(xref:spec-metamodel/common.adoc#semanticId[Submodel/semanticId]) they shall differ in their ((xref:spec-metamodel/common.adoc#version[Submodel/administration]))

a) xref:spec-metamodel/common.adoc#version[version]   (xref:spec-metamodel/common.adoc#revision[revision] is ignored) 

> or in their

Reviewer: ```suggestion
or 
```
"their" is now duplicated with the following bullet point. Suggestion: keep as it was before, i.e., only use `or`

b) their xref:spec-metamodel/common.adoc#createdAt[createdAt] or xref:spec-metamodel/common.adoc#updatedAt[updatedAt] date

> or in their

Reviewer: ```suggestion
or 
```
"their" is now duplicated with the following bullet point. Suggestion: keep as it was before, i.e., only use `or`

c) their xref:spec-metamodel/common.adoc#creator[creator]  

> or in their

Reviewer: ```suggestion
or 
```

d) their xref:spec-metamodel/common.adoc#supplementalSemanticId[supplementalSemanticId]

> or in their

Reviewer: ```suggestion
or 
```

e) xref:spec-metamodel/common.adoc#templateId[templateId] 

====

> Note: per specification the xref:spec-metamodel/common.adoc#Identifiable[IDs] of these two Submodels are not identical.

Reviewer: ```suggestion
Note: Per specification the xref:spec-metamodel/common.adoc#Identifiable[IDs] of any two submodels in an Asset Administration Shell can never be identical.
```
